src/Umbraco.Cms.Integrations.Automation.Zapier/Helpers/FormsHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Umbraco.Cms.Integrations.Automation.Zapier.Helpers
+{
+    public class FormsHelper
+    {
+        public static bool IsFormsExtensionInstalled => Type.GetType("Umbraco.Forms.Core.Services.IFormService, Umbraco.Forms.Core") != null;
+
+        public static IEnumerable<MethodInfo> GetMethodsForType(Type type)
```

Comment:
I'd suggest moving this method and the one below into a new class, e.g. `ReflectionHelper`, just as they don't really have anything to do with Forms and are more utility methods.

Reply:
Updated.